src/sentry/integrations/gitlab/issues.py

```diff
@@ -35,13 +44,9 @@ def create_default_repo_choice(self, default_repo):
         return (project['id'], project['name_with_namespace'])
 
     def get_create_issue_config(self, group, **kwargs):
-        params = kwargs.get('params', {})
-        defaults = self.get_project_defaults(group.project_id)
-        kwargs['repo'] = params.get('project', defaults.get('project'))
```

Comment:
Is the `repo` key needed in `get_create_issue_config()`?

Reply:
Yes. So this was what I did to make it work with the default's code here: https://github.com/getsentry/sentry/blob/master/src/sentry/integrations/issues.py#L198:L226

Reply:
Should the code be re-ordered then? The `repo` key isn't set until after `get_create_issue_config()` is called now.